tests/test_visitors/test_ast/test_imports/test_vague_imports.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    VagueImportViolation,
+)
+from wemake_python_styleguide.visitors.ast.imports import WrongImportVisitor
+
+import_template = 'from mod import {0}'
+
+blacklisted_method_names = [
+    'dump',
+    'dumps',
+    'load',
+    'loads',
+    'parse',
+    'safe_dump',
+    'safe_dump_all',
+    'load_all',
+    'dump_all',
+    'safe_load_all',
+    'safe_dump_all',
+]
+
+
+@pytest.mark.parametrize(
```

Comment:
Can you please also test multiple imports like: `from some import parse, dumps`
And named imports like: `from some import dumps as json_dumps`

Reply:
Actually, it is an interesting question: should named imports of the vague names trigger a violation? IMO they should not, as long as alias conform to the same rules.

Reply:
> they should not

I agree 👍

Reply:
Yes, `from json import loads as json_loads` is silly, but is ok for this check :+1: